src/audio/CMakeLists.txt

```diff
@@ -112,15 +115,16 @@ check_optimization(fma -mfma -DOPS_FMA)
 check_optimization(hifi2ep -mhifi2ep -DOPS_HIFI2EP)
 check_optimization(hifi3 -mhifi3 -DOPS_HIFI3)
 
-set(sof_audio_modules volume src asrc eq-fir eq-iir dcblock)
+set(sof_audio_modules volume src asrc eq-fir eq-iir dcblock crossover)
 
 # sources for each module
 set(volume_sources volume/volume.c volume/volume_generic.c)
 set(src_sources src/src.c src/src_generic.c)
 set(asrc_sources asrc/asrc.c asrc/asrc_farrow.c asrc/asrc_farrow_generic.c)
 set(eq-fir_sources eq_fir/eq_fir.c eq_fir/fir.c)
-set(eq-iir_sources eq_iir/eq_iir.c eq_iir/iir.c eq_iir/iir_generic.c)
```

Comment:
this method of moving into math still worries me because won't we need to move the crossover into the math sections as well once we start building the multiband DRC?

Reply:
I am noticing that all the libraries are linked back to SOF but not cross-linked, I wonder if we need need to add a case for cross linking libs

Reply:
@singalsu can you give us your 2c here on the maths library, which functions still need to go in it. Do you have a rough plan of how it should look ? This should mostly be code moving, or is there some coupling that prevents a code move today ?